mailbox/api/src/main/java/org/apache/james/mailbox/model/SearchQuery.java

```diff
@@ -1093,7 +1104,9 @@ public String toString() {
         TEXT,
 
         /** Full message content including headers */
```

Comment:
+ Attachment if I read code correctly. That's weird.

Reply:
WDYM ?

Reply:
I mean it includes Attachment too